src/status_im/contexts/chat/messenger/messages/content/emoji/view.cljs

```diff
@@ -0,0 +1,10 @@
+(ns status-im.contexts.chat.messenger.messages.content.emoji.view
+  (:require [react-native.core :as rn]
+            [status-im.contexts.chat.messenger.messages.content.emoji.style :as style]))
+
+(defn emoji-message
```

Comment:
nit, can we change this to `view` - the ns should be emoji-message.view.

then when you use it, it will be.
`[emoji-message/view ...]`